types/three/examples/jsm/webxr/XRHandOculusMeshModel.d.ts

```diff
@@ -0,0 +1,23 @@
+import { Group, Object3D } from './../../../src/Three';
+
+import { XRHandModel } from './XRHandModelFactory';
+
+export interface XRHandOculusMeshModelOptions {
+    model: 'lowpoly';
+}
+
+export class XRHandOculusMeshModel {
+    controller: Group;
+    handModel: XRHandModel;
+    bones: Array<Object3D | null>;
+
+    constructor(
+        handModel: XRHandModel,
+        controller: Group,
+        path: string,
+        handedness: 'left' | 'right',
```

Comment:
Can we make this a type (e.g., `type Handedness = 'left' | 'right';`) that can be shared between the two usages of it?

Reply:
We could, im not sure the best place to put it though? these are `examples` so they're used in isolation...

Reply:
Either a new file that can be shared between the two (e.g., `XRHandModelCommon`) or in one of the factory files (e.g., `XRControllerModelFactory` or `XRHandModelFactory`) since it seems like the normal use case to create a hand model using the factory classes.

Reply:
Lets put it in `XRHandModelCommon` I want to avoid *more* files